cmd/buildah/manifest.go

```diff
@@ -155,6 +156,19 @@ func init() {
 	manifestRemoveCommand.SetUsageTemplate(UsageTemplate())
 	manifestCommand.AddCommand(manifestRemoveCommand)
 
+	manifestExistsCommand := &cobra.Command{
+		Use:   "exists MANIFEST",
```

Comment:
Other `Use` fields for `manifest` commands don't list arguments here, so this shouldn't.